bayes3d/viz/viz.py

```diff
@@ -62,9 +62,10 @@ def get_depth_image(image, max=None):
         maxim = max
     mask = depth < maxim
     depth[np.logical_not(mask)] = np.nan
-    vmin = depth[mask].min()
-    vmax = depth[mask].max()
-    depth = (depth - vmin) / (vmax - vmin)
+    if np.sum(mask) > 0:
```

Comment:
theres one more fix that i made locally, maybe you can think about it and incorporate it
```
 cmap  = copy.copy(plt.get_cmap('turbo'))
 cmap.set_bad(color=(1.0, 1.0, 1.0, 1.0))
 
-def get_depth_image(image, max=None):
+def get_depth_image(image, max=None, remove_max=True):
     """Convert a depth image to a PIL image.
     
     Args:
@@ -58,6 +58,8 @@ def get_depth_image(image, max=None):
     depth = np.array(image)
     if max is None:
         maxim = depth.max()
+        if not remove_max:
+            maxim += 1
     else:
         maxim = max
     mask = depth < maxim
```

Reply:
updated in a38afb881c8deaf8b8c73448566b8e5a4eea5df1